api/Window.json

```diff
@@ -1286,6 +1286,58 @@
           }
         }
       },
+      "deviceorientation_event": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Window/deviceorientation_event",
+          "description": "<code>deviceorientation</code> event",
+          "support": {
+            "chrome": {
+              "version_added": "7"
+            },
+            "chrome_android": {
+              "version_added": "18"
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "3.6"
```

Comment:
On the original compatibility table, Firefox has a note that says "Firefox 3.6, 4, and 5 supported [mozOrientation](https://developer.mozilla.org/en-US/docs/Web/Events/MozOrientation) versus the standard `DeviceOrientation` event.".  I think that we should bump this and `firefox_android` up to `6`, based upon the following message on `mozOrientation`'s page:

> This experimental API was removed in Gecko 6.0 (Firefox 6.0 / Thunderbird 6.0 / SeaMonkey 2.3), when support for the standard DeviceOrientationEvent was implemented. You should use that API instead.

I was thinking that we should also add an entry with these version numbers and `"alternative_name": "mozOrientation"`.